scripts/gm/bin/lib-gm

```diff
@@ -732,36 +760,39 @@ EOF
     RPC="$(get_rpc_port "$i")"
     GRPC="$(get_grpc_port "$i")"
     ID="$(get_chain_id "$i")"
+    ACCOUNT_PREFIX="$(get_wallet_account_prefix "$i")"
+    DENOM="$(get_staking_denom "$i")"
     cat <<EOF >> "$GLOBAL_HERMES_CONFIG"
 [[chains]]
 id='${ID}'
 rpc_addr='http://localhost:${RPC}'
 grpc_addr='https://localhost:${GRPC}'
 websocket_addr='ws://localhost:${RPC}/websocket'
 rpc_timeout='1s'
-account_prefix='cosmos'
+account_prefix='${ACCOUNT_PREFIX}'
 key_name='wallet'
 store_prefix='ibc'
-fee_denom='stake'
+fee_denom='${DENOM}'
 fee_amount=1000
-gas=300000
+gas=500000
 clock_drift='5s'
 trusting_period='14days'
 
 [chains.trust_threshold]
-numerator = '1'
-denominator = '3'
+numerator='1'
+denominator='3'
 
 EOF
     done
 }
 
 # This is a helper function that extracts the coinType from an absolute hdpath (m/44'/118'/0'/0/0) for hermes.
+# This will be obsolete after https://github.com/informalsystems/ibc-rs/issues/1049 .
```

Comment:
Is it obsolete now?

Reply:
It will be obsolete in 0.0.6 and I'll remove it there.